### DISCUSSION TITLE
SONOFF Zigbee 3.0 USB Dongle Plus V2 (E type)

### DISCUSSION THREAD

**Discussion by Domotixz** [Koenkk/zigbee2mqtt#15002] [Devices]
When will the SONOFF Zigbee 3.0 USB Dongle Plus V2 E support Touchlink?

**Comment by kirovilya** [comment]
already supports since 1.26.0

**Comment by Domotixz** [comment]
You are talking about the ITead Sonoff Zigbee 3.0 USB Dongle Plus V2 model "ZBDongle-E"? On the official zigbee2mqtt website is says that it is not supported and when searching in the frontend for Touchlink devices i get no results. In my humble opinion i would say it does not work. That's why i asked when it would be supported.

**Comment by kirovilya** [comment]
:) sorry, I forgot. you need to update the firmware of the stick https://github.com/Koenkk/zigbee2mqtt/discussions/13893

**Reply by Domotixz** [reply]
First of all. Tnx for the quick reply. When i am not mistaking the most recent firmware for this E-dongle is 6.10.3. (https://github.com/xsp1989/zigbeeFirmware/tree/master/firmware/Zigbee3.0_Dongle-NoSigned/EZSP)

When i take a look at my settings screen (about) i see => Coordinator revision 6.10.3.0 build 297

So, this would mean that i already have the latest firmware. right? But i have no touchlink support as far as i think. In the front-end i go to touchlink and run the scan. Result => Detected 0 touchlink devices. The devices i have are the following:

5 Aubess Mini Smart Switches. Recognized as Tuya WHD02. 
1 IKEA Shortcut button (E1812). 
1 IKEA motion detect (E1525/E1745). 
3 MOES MS-108ZR Blinds controllers 
2 IKEA power plugs (E1603/E1702/E1708). 

I would have expected that it would be possible to connect the shortcut button with touchlink to a powerplug so that they communicate directly and without the need of the coordinator. I have also many z-wave devices and they can be associated which is more or less the same feature. They can communicate directly without the need of the z-wave controller. Perhaps i have it all wrong or the devices simply don't support touchlink. Any clarification on this would be great.

**Reply by kirovilya** [reply]
in this case, the firmware version does not match the functionality. there is no touchlink in the official firmware from Sonoff. so you can flash third-party firmware with the same version, but with available touchlink functionality

**Reply by Domotixz** [reply]
Tnx again.

**Comment by kirovilya** [comment]
https://github.com/Koenkk/zigbee-herdsman/issues/319#issuecomment-1241133124
